src/comp_loinc/schema/part_schema.yaml

```diff
@@ -13,12 +13,12 @@ default_curi_maps:
 classes:
   PartClass:
     is_a: Thing
-    slots:
+    slots:  # fields in LoincPartLink_Primary.csv (except for subClassOf:
       - subClassOf
-      - part_number
-      - part_type
-      - part_name
-      - part_display_name
+      - part_number  # PartNumber, e.g. LP7289-4
+      - part_type  # PartTypeName / Property
+      - part_name  # LongCommonName? e.g. "R wave duration in lead AVR"
+      - part_display_name  # PartName? e.g. "Heart"
```

Comment:
## Schema comments
- [ ] Make sure these comments are correct

I'm just trying to understand how the LinkML model relates to the inputs.

<details><summary>Screenshot</summary>
<p>

<img width="948" alt="Screenshot 2024-07-28 at 6 59 59 PM" src="https://github.com/user-attachments/assets/844a496e-4199-46c9-a7cc-fe6476397fdb">


</p>
</details>